internal/server/data/identity.go

```diff
@@ -97,6 +97,23 @@ func ListIdentities(db *gorm.DB, selectors ...SelectorFunc) ([]models.Identity,
 	return list[models.Identity](db, selectors...)
 }
 
+func ListIdentitiesByGroup(db *gorm.DB, groupID uid.ID, selectors ...SelectorFunc) ([]models.Identity, error) {
+	group, err := GetGroup(db.Preload("Identities", func(db *gorm.DB) *gorm.DB {
+		for _, selector := range selectors {
```

Comment:
This adds the `name=?` and `ids=?` selectors to the query, and then sorts the results alphabetically to more closely resemble `ListIdentities()`